src/models/dependency-injection/controller.service.ts

```diff
@@ -34,13 +33,9 @@ export class ControllerService {
     return this.instances;
   }
 
-  async initControllers(options: ServerOptions): Promise<Application> {
+  async initControllers(application: Application): Promise<Application> {
```

Comment:
The output of this method could be `Promise<void>` as it only changes the instance of `application` that is passed.